src/components/menu/menu.js

```diff
@@ -264,6 +264,7 @@ class MenuItem extends Component {
         onClick={this.onClick}
         onKeyDown={this.onKeyDown}>
         <span>{props.data.label}</span>
+        <span>{props.data.badge ? props.data.badge : null}</span>
```

Comment:
If think the span is redundant. If it needed to be wrapped then wrap it with fragment. But I think this should also work:
```suggestion
        {props.data.badge ? props.data.badge : null}
```